src/System.Security.Cryptography.X509Certificates/tests/CertificateCreation/CertificateRequestChainTests.cs

```diff
@@ -179,6 +179,7 @@ public static void ChainCertRequirements(bool useIntermed, bool? isCA, X509KeyUs
                     chain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
                     chain.ChainPolicy.VerificationFlags = X509VerificationFlags.AllowUnknownCertificateAuthority;
                     chain.ChainPolicy.ExtraStore.Add(rootCert);
+                    chain.ChainPolicy.VerificationTime = start.ToLocalTime().DateTime;
```

Comment:
I think we'll be OK:

```C#
DateTimeOffset dto = new DateTimeOffset(2017, 11, 5, 8, 30, 15, TimeSpan.Zero);
DateTimeOffset dtoLocal = dto.ToLocalTime();
DateTime dtLocal = dtoLocal.DateTime;
DateTime dtUtc = dtLocal.ToUniversalTime();

Console.WriteLine(dto);
Console.WriteLine(dtoLocal);
Console.WriteLine(dtLocal);
Console.WriteLine(dtUtc);
```

Emits
```
11/5/2017 8:30:15 AM +00:00
11/5/2017 1:30:15 AM -07:00
11/5/2017 1:30:15 AM
11/5/2017 9:30:15 AM
```

So that's "fall back" (the UTC -> Local -> eventual UTC conversion went forward an hour).  The very last instance of this fixed will be on an edge case for that, which will happen to work out by quirky behavior.  Windows does \<= NotAfter, so it matches.  OpenSSL does \< NotAfter, but uses local time, so it might not have the "off by an hour" forward conversion.  Either way, I can bump the test from 1 hour aperture to 2, and save the ambiguity.

Rolling the start time forward one hour to 0930Z results in an unambiguous roundtrip.

"spring forward" is an unambiguous roundtrip, since there's a local time that doesn't exist, instead of a local time that overlaps.

---

I guess this analysis is a bit western hemispheric of me... I'd need to test on a machine where there's a positive UTC offset just to make sure it's not going to alter the results (and pick "an hour ago").